PV260_Project/Tests/BLTests/NotificationsTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using BusinessLayer.DiffComputing;
+using BusinessLayer.Notifications;
+using DataLayer.Models;
+using FluentAssertions;
+using NUnit.Framework;
+using BusinessLayer.Notifications;
+using Microsoft.Extensions.Options;
+using HtmlAgilityPack;
+
+using System.Linq;
+namespace Tests.BLTests;
+
+[TestFixture]
+public class NotificationsTests
+{
+    private List<HoldingChanges> _changes = new List<HoldingChanges>();
```

Comment:
IDE offers that it is redundant type specification .. but I think it's ok
same in _mailSettings